doc/articles/getting-started-tutorial-1.md

```diff
@@ -243,7 +243,7 @@ The tutorial walks you through creating a cross platform application with Uno Pl
     ```
 
     > [!IMPORTANT]
```

Comment:
"Important" does not display correctly in some parts of the file (and in some other file).
[maybe a character/format issue or a Viewer bug]
![image](https://user-images.githubusercontent.com/16295702/86629147-f7b52700-bf98-11ea-9b28-d23af53b6bc1.png)

Reply:
Good catch @agneszitte-nventive ! Indeed, the second 'IMPORTANT' in the screenshot was misformatted.

The `[!IMPORTANT]` is a docfx directive, it doesn't look nice in rendered Markdown unfortunately, but it produces special styling in the generated html https://platform.uno/docs/articles/getting-started-tutorial-1.html